Analysis/src/TauUncertainties.cpp

```diff
@@ -21,12 +21,14 @@ namespace analysis {
                                         {10, PhysicalValue(-1.2,0.8)} } }
         };
 
-        std::pair<double,double> correction_factor;
-        PhysicalValue tau_correction = tau_correction_factor.at(period).at(decayMode);
-        double result = 1 + ((tau_correction.GetValue() + static_cast<int>(scale) * tau_correction.GetStatisticalError())/100);
-        double uncertainty = 1 + ((static_cast<int>(scale) * tau_correction.GetStatisticalError())/100);
-        correction_factor.first = result;
-        correction_factor.second = uncertainty;
+        double correction_factor;
+        if(pt > 400)
+            correction_factor = 1 + static_cast<int>(scale) * 0.03;
+        else{
+            PhysicalValue tau_correction = tau_correction_factor.at(period).at(decayMode);
```

Comment:
check if bothe period and decayMode are present in the map, otherwise, throw an exception